starknet-foundry/crates/cast/tests/e2e/multicall/multicall_new.rs

```diff
@@ -0,0 +1,95 @@
+use crate::helpers::fixtures::default_cli_args;
```

Comment:
nitpick - perhaps file name could be just `new.rs`, to match the subcommand name? the same goes for run

Reply:
[rename](https://github.com/foundry-rs/starknet-foundry/pull/98/commits/f88c1c4491cdd7c7172bce01ad15a0fe6cba7acc)